sdk/eventhub/event-hubs/CHANGELOG.md

```diff
@@ -1,15 +1,11 @@
 # Release History
 
-## 5.10.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 5.11.0 (2023-06-06)
 
 ### Other Changes
 
+- Use Rhea's prefetch window to prefetch events from the service. This improves the performance of the receiver by reducing the number of round trips to the service. The default prefetch window is 3 * `maxBatchSize` events. This can be configured by setting the `prefetchCount` option on the `subscribe` method on `EventHubConsumerClient`.
```

Comment:
what happens when the user closes the consumer/subscription as soon as the prefetch receives the events?

Reply:
The receiver will be discarded including the prefetched list of events. This is ok in Event Hubs because when you receive from a partition, you start reading from a specific event position and not necessarily from the last event fetched.

Reply:
If no event position is specified, the reading will start from the earliest event in the partition.